src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
 public class SalaryInfo {
```

Comment:
Think of a different approach. Don't try to overcomplicate with lots of classes. You can solve it within SalaryInfo.